cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -1151,6 +1167,11 @@ def _deep_check(self):
                 "adata is loaded in backed mode, validation requires anndata to be loaded in memory"
             )
 
+        # Checks for deprecated columns
+        self._check_deprecated_columns()
+        if self.errors:
+            return
```

Comment:
I don't think we want this check + return here, the current pattern is collecting the various validation errors in one pass and printing all the error msgs in self.errors at the top-level call

Reply:
Good point, this was actually something I added temporarily. Removing it.